app/components/icons/library.tsx

```diff
@@ -84,8 +84,8 @@ export function ItemsIcon(props: SVGProps<SVGSVGElement>) {
 export function SettingsIcon(props: SVGProps<SVGSVGElement>) {
   return (
     <svg
-      width="100%"
-      height="100%"
+      width="20"
```

Comment:
Hmm shouldnt change it. So the way it works is that if width and height are 100% then it allows us to have the icon responsive based on the wrapper size. So I usually wrap it in a span and set the width and height there.